docs/content/api-reference/anthropic.md

```
@@ -0,0 +1,828 @@
+| GET | `/olla/anthropic/v1/models` | List available models in Anthropic format |
+| POST | `/olla/anthropic/v1/messages` | Create a message (chat completion) |
+| POST | `/olla/anthropic/v1/messages/count_tokens` | Estimate token count for a message |
+
+## POST /olla/anthropic/v1/messages
+
+Create a message with support for streaming, tool use, and vision.
+
+### Request
+
+**Headers**:
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)